packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js

```diff
@@ -216,5 +230,6 @@ InteractionLayer.propTypes = {
   width: PropTypes.number.isRequired
 }
 
-export default InteractionLayer
+export default withStores(InteractionLayer, storeMapper)
```

Comment:
I think `multiIimageCloneMarkers` should simply be passed to InteractionLayer from its container rather than hooking up the store in this file. InteractionLayerContainer's purpose is to grab MST values and pass them to its children, so it's not worth breaking that pattern in this PR.

Also [withStores](https://github.com/zooniverse/front-end-monorepo/tree/master/packages/lib-classifier/src/helpers/withStores) is an outdated method for Class components. New connections to the MST store should have [useStores](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/hooks/useStores.js) instead.

Reply:
Good point on this. I pulled all of the code around this out and instead am passing the `multiImageCloneMarkers` prop from the `InteractionLayerContainer` to the `InteractionLayer`. 

Noted on useStores vs withStores going forward.